package.json

```diff
@@ -9,14 +9,24 @@
     "url": "https://github.com/the-norman-sicily-project/interactive-map.git"
   },
   "dependencies": {
+    "leaflet": "^1.3.4",
+    "leaflet.markercluster": "^1.4.0",
+    "prop-types": "^15.6.2",
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
+    "react-leaflet": "^2.0.1",
+    "react-leaflet-markercluster": "^2.0.0-rc3",
     "react-scripts": "1.1.5"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "devDependencies": {
```

Comment:
Missing dependencies for eslint:

```
warning " > eslint-config-airbnb@17.1.0" has unmet peer dependency "eslint@^4.19.1 || ^5.3.0".
warning " > eslint-config-airbnb@17.1.0" has unmet peer dependency "eslint-plugin-import@^2.14.0".
warning " > eslint-config-airbnb@17.1.0" has unmet peer dependency "eslint-plugin-jsx-a11y@^6.1.1".
warning " > eslint-config-airbnb@17.1.0" has unmet peer dependency "eslint-plugin-react@^7.11.0".
warning "eslint-config-airbnb > eslint-config-airbnb-base@13.1.0" has unmet peer dependency "eslint@^4.19.1 || ^5.3.0".
warning "eslint-config-airbnb > eslint-config-airbnb-base@13.1.0" has unmet peer dependency "eslint-plugin-import@^2.14.0".
```

Reply:
Realized that for some reason they aren't in the commit but are in my local. Would you know a reason for why this would occur?

Reply:
Is it possible you added them after you committed the file? Or was the file not saved?

Reply:
Is it possible you added them after you committed the file? Or was the file not saved?

Reply:
There were added before. Otherwise, VSCode would allow me to stage it right now. The file and working tree are the same. DevDependencies were not there until I added the ESLint stuff for Airbnb and prettier. For some reason, it committed that object as blank? or so it would seem.

Reply:
Weird...

Reply:
Added some of the things it said it was missing. The missing ones from the file are now there and not showing as an add. I am getting an error myself though. I can't add ESLint to the dependencies because its part of create-react-app so it throws an error. Probably don't want to eject as well.